modules/clients/src/test/java/org/apache/ignite/jdbc/thin/JdbcThinMetadataSelfTest.java

```diff
@@ -203,6 +206,82 @@ public void testResultSetMetaData() throws Exception {
         assert "java.lang.Integer".equals(meta.getColumnClassName(2));
     }
 
+    /**
+     * Ensure metadata returned for cache without explicit query entities:
+     *      - create a cache with indexed types and without explicit query entities
+     *      - request metadata for columns from the cache
+     *      - verify returned result
```

Comment:
It is not correct javadoc "list", it should be written in HTML style like
`<ul>
<li> .... </li>
</ul>`
So, it is a test though... who cares.